### PR TITLE
refactor(rust): integrate uppercase and echoer workers to nodemanager

### PR DESCRIPTION
**Pull request by leonzchang**
move uppercase and echoer workers to node manager as services. #3056

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
@leonzchang thank you for sending this 🙏 
We can merge after @SanjoDeundiak / @adrianbenavides have a chance to review.

**Comment by leonzchang** [comment]
@mrinalwadhwa sure, no problem. 😀

File: implementations/rust/ockam/ockam_api/src/nodes/models/services.rs [hunk 1]
```diff
@@ -63,3 +63,43 @@ impl<'a> StartAuthenticatedServiceRequest<'a> {
...
     }
 }
+
+/// Request body when instructing a node to start an Uppercase service
+#[derive(Debug, Clone, Decode, Encode)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct StartUppercaseServiceRequest<'a> {
+    #[cfg(feature = "tag")]
+    #[n(0)] tag: TypeTag<5179596>,
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_api/src/nodes/models/services.rs]: These numbers should be unique. You can use this page to generate a valid TypeTag: https://www.random.org/integers/?num=100&min=1000000&max=9999999&col=5&base=10&format=html&rnd=new

File: implementations/rust/ockam/ockam_api/src/nodes/models/services.rs [hunk 2]
```diff
@@ -63,3 +63,43 @@ impl<'a> StartAuthenticatedServiceRequest<'a> {
...
+    }
+}
+
+/// Request body when instructing a node to start an Echoer service
+#[derive(Debug, Clone, Decode, Encode)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct StartEchoerServiceRequest<'a> {
+    #[cfg(feature = "tag")]
+    #[n(0)] tag: TypeTag<5179596>,
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_api/src/nodes/models/services.rs]: Same here.

**Review by adrianbenavides — CHANGES_REQUESTED** [review]
Looks good to me! Just a small detail regarding `TypeTag`.

**Comment by leonzchang** [comment]
Oops, I didnt notice that. @adrianbenavides  thank you for the review.

File: implementations/rust/ockam/ockam_api/src/nodes/models/services.rs (hunk 2 above)
Comment (leonzchang) [reply implementations/rust/ockam/ockam_api/src/nodes/models/services.rs]: fix in afb3e03

File: implementations/rust/ockam/ockam_api/src/nodes/models/services.rs (hunk 1 above)
Comment (leonzchang) [reply implementations/rust/ockam/ockam_api/src/nodes/models/services.rs]: fix in afb3e03

**Review by adrianbenavides — DISMISSED** [review]
Thanks @leonzchang! I've forced-pushed your last commit fixing the commit message to pass the linter.

**Review by mrinalwadhwa — DISMISSED** [review]

**Review by mrinalwadhwa — APPROVED** [review]

**Pull request merged**